snakemake/logging.py

```
@@ -305,12 +243,504 @@
+
+
+def format_dict(dict_like, omit_keys=None, omit_values=None):
+    from snakemake.io import Namedlist
+
+    omit_keys = omit_keys or []
+    omit_values = omit_values or []
+
+    if isinstance(dict_like, Namedlist):
+        items = dict_like.items()
+    elif isinstance(dict_like, dict):
+        items = dict_like.items()
```

Ruff

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)